docs/configuration/operator-args.rst

```diff
@@ -47,12 +47,10 @@ dbt-related
 - ``dbt_cmd_flags``: List of command flags to pass to ``dbt`` command, added after dbt subcommand
 - ``dbt_cmd_global_flags``: List of ``dbt`` `global flags <https://docs.getdbt.com/reference/global-configs/about-global-configs>`_ to be passed to the ``dbt`` command, before the subcommand
 - ``dbt_executable_path``: Path to dbt executable.
-- ``env``: Declare, using a Python dictionary, values to be set as environment variables when running ``dbt`` commands.
```

Comment:
Since our docs are not versioned, it may be worth keeping these - and just explain they were deprecated here, WDYT?